index.js

```diff
@@ -5,46 +5,39 @@ module.exports = async (req, res, config) => {
 	const db = new (require("@replit/database"))();
 	const retronid = require("retronid");
 	const ejs = require("ejs");
+	const url = new URL("http://" + req.host + req.url);
+	const urlmdle = require("url");
+	var queryString = urlmdle.parse(req.url, true).query;
 	const regx = new RegExp(/\/key\/.+/g);
-	const renderFile = function (req) {
-		const { url } = req.query;
+	const regxcloud = new RegExp(/\/cloudkey\/\?key=.+/g);
+	const { usernamee, projectid, commentid } = config;
+	const renderFile = function (queryString, cloud) {
+		const { url } = queryString;
 		let text,
 			code = "";
 		try {
 			if (url !== undefined) {
 				code = retronid.generate();
-				ejs.renderFile(
-					"./pages/authwithcode.ejs",
-					{ code: code, url: url },
-					{},
-					function (err, str) {
-						if (err) {
-							throw err;
-						}
-						text = str;
-					},
-				);
+				ejs.renderFile(`./pages/authwith${cloud}.ejs`, { code: code, url: url }, {}, function (err, str) {
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/onedotprojects/scratchCommentAuth/pullRequest?prid=7364619)
```suggestion
				ejs.renderFile(`./pages/authwith${cloud}.ejs`, { code, url: url }, {}, function (err, str) {
```

Reply:
i never said that that was a bot

Reply:
that isn't the issue, this issue is the pages that get the key return empty for some dumb reason.